prody/proteins/interactions.py

```diff
@@ -3712,7 +3728,10 @@ def calcLigandInteractionsTrajectory(self, atoms, trajectory=None, filename=None
                 
                 ligs_per_frame_interactions = []
                 for ligs in ligand_interactions:
-                    LP_interactions = listLigandInteractions(ligs) 
+                    if output == 'info':
```

Comment:
you can just do `LP_interactions = listLigandInteractions(ligs, output=output)` without the if block

Reply:
also do you need to pop it and provide it again at all? Can you not just keep it in kwargs and have
`LP_interactions = listLigandInteractions(ligs, **kwargs)`?

Reply:
I removed filename=None, but I left trajectory=None.